cluster/juju/layers/kube-dns/README.md

```diff
@@ -0,0 +1,16 @@
+# KubeDNS
+
+Deploy a Kubernetes DNS server attached to any unit located in the cluster.
+
+## Usage
+
+```
+juju deploy kube-dns
+juju add-relation kube-dns mycharm
+```
+
+## More Information
+
+[Upstream Documentation](https://kubernetes.io/docs/admin/dns)
+
+
```

Comment:
You will need more of a readme than this. It is ok for now, but the readme is really short.